src/python/WMCore/WMSpec/StdSpecs/StepChain.py

```diff
@@ -469,7 +469,8 @@ def getWorkloadCreateArgs():
                     "FirstEvent": {"default": 1, "type": int, "validate": lambda x: x > 0,
                                    "null": False},
                     "FirstLumi": {"default": 1, "type": int, "validate": lambda x: x > 0,
-                                  "null": False}
+                                  "null": False},
+                    "ParentageResolved": {"default": False, "type": bool, "null": False},
```

Comment:
This should never be updated using web ui or manually set by user. So I thought it is in  a way safer to not to use strToBool to prevent user update. However, I am not sure whether that cause the problem down the line. If you prefer to change strToBool, I will do that.